default-plugins/strider/src/main.rs

```diff
@@ -24,37 +30,66 @@ impl ZellijPlugin for State {
                     *self.selected_mut() = min(self.files.len() - 1, next);
                 }
                 Key::Right | Key::Char('\n') | Key::Char('l') if !self.files.is_empty() => {
-                    match self.files[self.selected()].clone() {
-                        FsEntry::Dir(p, _) => {
-                            self.path = p;
-                            refresh_directory(self);
-                        }
-                        FsEntry::File(p, _) => open_file(&p),
-                    }
+                    self.traverse_dir_or_open_file();
+                    self.ev_history.clear();
                 }
                 Key::Left | Key::Char('h') => {
                     self.path.pop();
                     refresh_directory(self);
                 }
-
                 Key::Char('.') => {
                     self.toggle_hidden_files();
                     refresh_directory(self);
                 }
 
                 _ => (),
-            };
+            },
+            Event::Mouse(mouse_event) => match mouse_event {
+                Mouse::ScrollDown(_) => {
+                    let next = self.selected().saturating_add(1);
+                    *self.selected_mut() = min(self.files.len() - 1, next);
```

Comment:
This crashes if you try to scroll down in an empty folder :)
```
DEBUG  |strider                  | 2021-09-06 14:01:16.548 [id: 1     ] thread 'main' panicked at 'attempt to subtract with overflow', default-plugins/strider/src/main.rs:50:48 
DEBUG  |strider                  | 2021-09-06 14:01:16.548 [id: 1     ] note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace 
```
A `saturating_sub` with the `len` should fix things :)